### PR TITLE
Restore reference to Unique Identifier

### LINKED ISSUE #1894: RS spec section on unique-identifier should talk about dc:identifier instead/as well
https://w3c.github.io/epub-specs/epub33/rs/#sec-pkg-doc-pub-identifiers says:

> Reading Systems SHOULD NOT depend on the `unique-identifier` attribute being unique to one and only one EPUB Publication. 

This sounds like it's referring to `<package ... unique-identifier="pub-id">` , not `<dc:identifier id="pub-id">confreq-rs-pkg-unique-id</dc:identifier>`. But in practice, `unique-identifier` is basically never unique while `dc:identifier` usually is.

This is better in the format spec (https://w3c.github.io/epub-specs/epub33/core/#elemdef-opf-dcidentifier), which says:
> The EPUB Creator MUST provide an identifier that is unique to one and only one EPUB Publication — its Unique Identifier — in an `identifier` element. This `identifier` element MUST specify an `id` attribute whose value is referenced from the package element's `unique-identifier` attribute.

I think the RS spec should be edited to make clear that we're mostly talking about `dc:identifier`, not `unique-identifier`. Or maybe both of them.

### PR DESCRIPTION
**Pull request by mattgarrish**
s/unique-identifier attribute/Unique Identifier/ in caution against reading systems depending on uniqueness.

Fixes #1894

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/pull/1895.html" title="Last updated on Nov 6, 2021, 4:36 PM UTC (d6a5d31)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/1895/fb98744...d6a5d31.html" title="Last updated on Nov 6, 2021, 4:36 PM UTC (d6a5d31)">Diff</a>

### CODE REVIEW

File: epub33/rs/index.html [hunk 1]
```diff
@@ -364,10 +364,11 @@ <h4>Base Direction</h4>
 				<h3>Unique Identifier</h3>
 
 				<p id="confreq-rs-pkg-unique-id" data-tests="#confreq-rs-pkg-unique-id">Reading Systems SHOULD NOT
-					depend on the <code>unique-identifier</code> attribute being unique to one and only one EPUB
-					Publication. Determining whether two EPUB Publications with the same Unique Identifier represent
-					different versions of the same publication, or different publications, may require inspecting other
-					metadata, such as the last modification date, titles, and authors.</p>
+					depend on the <a href="https://www.w3.org/TR/epub-33/#dfn-unique-identifier">Unique Identifier</a>
```
Comment (dlazin) [inline epub33/rs/index.html:367]: Here or somewhere else, could we link to https://www.w3.org/TR/epub-33/#sec-opf-dcidentifier? That section is much more explicit about the fact that the unique-identifier is not actually the Unique Identifier itself :)

**Review by dlazin — APPROVED** [review]

File: epub33/rs/index.html (hunk 1 above)
Comment (mattgarrish) [reply epub33/rs/index.html:367]: I've updated the definition for Unique Identifier to be more specific that it is the value of the dc:identifier specified by the unique-identifier attribute (and added linking). Let me know if this works better for you now.

**Review by iherman — APPROVED** [review]

File: epub33/rs/index.html (hunk 1 above)
Comment (dlazin) [reply epub33/rs/index.html:367]: Much better :)

**Review by dlazin — APPROVED** [review]

**Comment by dlazin** [comment]
Merging now to avoid conflict with my test-renaming mega-PR.

**Pull request merged**